cmd/loop/loopout.go

```diff
@@ -176,6 +190,18 @@ func loopOut(ctx *cli.Context) error {
 			ctx.Int64("max_swap_routing_fee"),
 		)
 	}
+
+	// Calculate max miner fee. The prepay amount will be used if not set.
+	maxMinerFee, err := calculateMaxMinerFee(ctx, amt)
+	if err != nil {
+		return err
+	}
+	if maxMinerFee == 0 {
+		maxMinerFee = limits.maxPrepayAmt
+	}
+	// Override the old maxMinerFee.
+	limits.maxMinerFee = maxMinerFee
```

Comment:
can be simplified to:
```
limits.maxMinerFee, err   = calculateMaxMinerFee(ctx, amt, limits.maxPrepay)
```

See comment below re passing prepay value into calcMaxFee